packages/nextjs/components/scaffold-eth/Input/IntegerInput.tsx

```diff
@@ -3,6 +3,7 @@ import { CommonInputProps, InputBase, IntegerVariant, isValidInteger } from "~~/
 
 type IntegerInputProps = CommonInputProps<string | bigint> & {
   variant?: IntegerVariant;
+  multiplyBy1e18Enabled?: boolean;
```

Comment:
[nit]: I personally prefer to work with pre-fixes for these boolean modifier flags. Makes it a bit clearer when reading the variable name (or when autocompleting props on a component) that it is a boolean flag. So something like `enableMultiplyBy1e18`?

Reply:
Hmmm (some further nitting, sorry :sweat_smile: ), since setting a flag to true can be done like this `<IntegerInput enableMultiplyBy1e18 />`, it might make more sense to make the default value `false`. Otherwise devs would need to write `<IntegerInput enableMultiplyBy1e18={false} />`. To keep the desired default behavior, we could invert the name into `disableMultiplyBy1e18`.

```
<IntegerInput /> // default
<IntegerInput disableMultiplyBy1e18 /> // hides the asterisk
```

Reply:
I hid that asterisk in some apps too. And since it hides suffix I just named prop `hideSuffix`. Also I like Samuel's variant `disableMultiplyBy1e18`

Reply:
@sverps @rin-st thanks for the feedback!! 

I changed the flag to disableMultiplyBy1e18 (I think it's more self-explanatory than hideSuffix).